propsigner/client.go

```diff
@@ -0,0 +1,66 @@
+package propsigner
```

Comment:
Here's a simple client interface to ask for a remote wallet to sign a deal proposal, just call `RequestSignatureV1` and will give you the signature you'll need.

This is the method that we'll use in `dealerd` to ask the remote wallet for deal proposal signatures.